### DISCUSSION TITLE
Keycloak Login

### DISCUSSION THREAD

**Discussion by shobeck** [cypress-io/cypress#14240] [CI setup]
I really want Cypress to work for me.  I have tried this keycloak recommendation but it still hasn't worked:

https://vrockai.github.io/blog/2017/10/28/cypress-keycloak-intregration/

Does anyone have an alternative to that approach that works?  I have also tried the NPM projects 'cypress-keycloak-commands' and 'cypress-keycloak'; still doesn't work.

Background: Angular front-end, Spring back-end, keycloak for authentication.  Nothing particularly unusual, standard stuff.

Thanks in advance.

**Comment by vishumane** [comment]
I am also facing exact same kinda issue.
only difference, frontend is ReactJS.
( the post request to keycloak/token endpoint successful, getting access_token )
Thanks in advance.
